recipes/uritemplate.py/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "uritemplate.py" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: 4dd14904ba502c6ff8d6276e004404de
+
+build:
+  number: 0
+  script: python setup.py install
```

Comment:
No worries. I just initially used `setuptools` based on what I saw in `master` and realized that didn't work with this version. Figured I'd make a note so that we remember to switch after the next release.